Source/WebCore/dom/make-event-names.py

```diff
@@ -209,7 +251,27 @@ def writeln(text):
             delimiter = ','
             if conditional:
                 writeln('#endif')
-
+        writeln(f'    {delimiter} m_typeInfoMap({{')
```

Comment:
In a subsequent change, we should make it tri-searching code instead (as the same to HTMLName etc.).

Reply:
You think that would be beneficial even though the string is already literal / AtomString?